.gitignore

```diff
@@ -0,0 +1,7 @@
+__pycache__/
+*.pyc
+.coverage
```

Comment:
.env

Reply:
I strongly suggest you add this to your global `.gitignore`. Everyone names his "env" folder differently, so I always have a bit of an issue adding it to `.gitignore`. It is such an endless battle :(